pyvista/core/grid.py

```diff
@@ -97,6 +87,9 @@ class RectilinearGrid(vtkRectilinearGrid, Grid):
 
     """
 
+    _READERS = {'.vtk': vtk.vtkRectilinearGridReader, '.vtr': vtk.vtkXMLRectilinearGridReader}
```

Comment:
Class attributes should be lowercase.

Reply:
Should they if they're constants though?

Reply:
Valid point.  Thought that constants were only defined at the module level and didn't consider it at the class level.  This cleared it up for me:
https://stackoverflow.com/a/45702782/3369879